### DISCUSSION TITLE
Help on setting up Alldebrid and realdebrid

### DISCUSSION THREAD

**Discussion by pjzarxc** [pyload/pyload#4277] [Help]
hi there is there any complete video or doc reference available to setup the ALLdebrid and realdebrid i always gets the error get the files
GetRealdebridToken.py
AlldebridCom.py
files but i dont know how to use them and how to setup kindly help. I know asking a noob question but i will be very thankfull if any one help.

**Comment by GammaC0de** [comment]
On any computer that has python installed download GetRealdebridToken.py, open terminal/cmd and run `python GetRealdebridToken.py` and follow the instructions.

**Reply by ahmed2m** [reply]
But after that how do I use the plugin/account. Mine says premium: yes but everything still goes through the "DefaultPlugin" or the host's plugin not realdebrid

**Reply by GammaC0de** [reply]
Maybe realdebrid does not support this URL, what URL did you try?

**Reply by ahmed2m** [reply]
Mainly mega[.]nz, but I remember others (also supported by realdebrid) also go through (host name) plugin not realdebrid

**Comment by mrtargaryen** [comment]
How do I add Alldebrid? I'm new to pyload today (installed in docker)  and when I add [https://github.com/pyload/pyload/blob/develop/src/pyload/plugins/downloaders/AlldebridCom.py](url) I get the following error:

`# python3 /home/server/Downloads/AlldebridCom.py
Traceback (most recent call last):
  File "/home/server/Downloads/AlldebridCom.py", line 7, in <module>
    from ..base.multi_downloader import MultiDownloader
ImportError: attempted relative import with no known parent package
`

**Reply by GammaC0de** [reply]
> How do I add Alldebrid?

You download and run `GetAlldebridTokenV4.py` from here: https://github.com/pyload/pyload/files/4489732/GetAlldebridTokenV4.zip to get you username and password for the Alldebrid plugin.

After that, you browse to pyload and go to settings -> accounts -> add the Alldebrid account.
Download.
